### PR TITLE
Add package.xml for PECL package

### PR DESCRIPTION
**Pull request by oschwald**
@remicollet, would you be willing to review this when you get a chance? I wrote this by hand. I am not sure if there is an specific tooling you would recommend.

Also, I noticed that running `pear package` rewrites the file. Would you recommend committing those changes? From looking at other people's repos, it appears some people do commit the `pear` version and others don't.

Thank you!

### CODE REVIEW

File: package.xml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+                    <file role="test" name="002-final.phpt"/>
+                    <file role="test" name="003-open-basedir.phpt"/>
+                </dir>
+            </dir>
+        </dir>
+    </contents>
+    <dependencies>
+        <required>
+            <php>
+                <min>7.2.0</min>
```
Comment (remicollet) [inline package.xml:53]: Version 1.6.0 is compatible with 5.6, right ?
Is this dependency bump expected with 1.7.0 ?

Comment (remicollet) [reply package.xml:53]: Forget, explained in the changelog

**Comment by remicollet** [comment]
> I am not sure if there is an specific tooling you would recommend.

no, only your prefrerred editor

> Also, I noticed that running pear package rewrites the file.

Yes, with the file checksum

> Would you recommend committing those changes? 

No

> From looking at other people's repos, it appears some people do commit the pear version ...

That doesn't make sense... 

Please also use "pecl package ..." (not "pear package ...")

File: package.xml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+            </php>
+            <pearinstaller>
+                <min>1.4.0dev13</min>
```
Comment (remicollet) [inline package.xml:56]: Can use 1.10.0 as this is the 1st version compatible with 7.x

**Comment by oschwald** [comment]
Thanks for the feedback! I've incorporated it and will do a release shortly.

File: package.xml [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+    <name>maxminddb</name>
+    <channel>pecl.php.net</channel>
+    <summary>Reader for the MaxMind DB file format</summary>
+    <description>This is the PHP extension for reading MaxMind DB files. MaxMind DB is a binary file format that stores data indexed by IP address subnets (IPv4 or IPv6).</description>
+    <lead>
+        <name>Greg Oschwald</name>
+        <user>oschwald</user>
+        <email>goschwald@maxmind.com</email>
+        <active>yes</active>
```
Comment (oschwald) [inline package.xml:15]: @remicollet, I must have done something wrong here. When I try to release, I get:

> You can not delete your own maintainer role or you will not be able to complete the update process. Set your name in package.xml or let the new lead developer upload the new release


I tried changing `<name>`  to `<name>Gregory Oschwald</name>`, which matches the information on the site, but that didn't help. I checked the other information and didn't see any discrepancies.

**Comment by remicollet** [comment]
AFAIK the "user" must match your login account... but everything seems ok
* https://pecl.php.net/package/maxminddb
* https://pecl.php.net/user/oschwald

Don't know what's wrong, please ask on the ML

**Comment by remicollet** [comment]
Discussed with Derick, please try again

**Comment by oschwald** [comment]
I am still getting it. I'll email the list. Thanks!

**Pull request merged**